### DISCUSSION TITLE
GitHub Runners not running, my $spend for the last several months is $0

### DISCUSSION THREAD

**Discussion by nicowilliams** [community/community#56505] [Actions]
### Select Topic Area

Bug

### Body

Ubuntu and Windows (not self-hosted) runners are not running on my repos/PRs.  I can't tell why.  GHA sits there waiting for a runner that is never assigned.  No self-hosted runners are configured.  I've not spent any GitHub runner minutes in some time.  For an example see https://github.com/heimdal/heimdal and my clone of it.

**Comment by airtower-luna** [comment]
I took a peek [at one of the failed workflows](https://github.com/heimdal/heimdal/actions/runs/5026392648/workflow#L60) and the problem is pretty obvious: The jobs are configured to use `ubuntu-18.04` runners. Those have been deprecated for a long time and were removed in April. You'll have to update to one of the [current runners](https://github.com/actions/runner-images) (`ubuntu-22.04` would make sense as it is the most recent one).

See: https://github.blog/changelog/2022-08-09-github-actions-the-ubuntu-18-04-actions-runner-image-is-being-deprecated-and-will-be-removed-by-12-1-22/

**Reply by nicowilliams** [reply]
It's not obvious at all in the UI.  I'm glad it's obvious to you and you could tell us, but please fix the UI.

**Reply by nicowilliams** [reply]
~It's also not obvious what the problem is re: Windows.~

**Reply by airtower-luna** [reply]
> but please fix the UI.

That's something GitHub would have to do, I can't. :wink:
